gen/calc.py

```diff
@@ -1312,7 +1308,10 @@ def calculate_fault_domain_detect_contents(fault_domain_detect_filename):
         'enable_mesos_ipv6_discovery': 'false',
         'log_offers': 'true',
         'mesos_cni_root_dir_persist': 'false',
-        'enable_mesos_input_plugin': 'true'
+        'enable_mesos_input_plugin': 'true',
+        'calico_enabled': 'true',
+        'calico_overlay_netwokr_cidr': '44.129.0.0/20',
+        'etcd_endpoints': 'http://master.mesos:2379'
```

Comment:
`etcd_endpoints` is for test only, and this will be changed when  ETCD is done.

Reply:
ETCD will have a broader usage, so the parameter has no `calico` prefix for the specification. For now, only calico use ETCD.